test/conformance/ptp/ptp.go

```diff
@@ -516,6 +516,18 @@ var _ = Describe("[ptp]", func() {
 })
 
 func configurePTP() {
+	/*
+		The test takes significant amount of time checking
+		for the right interfaces to test
+		By setting the following variable environment for test purposes
+		The suite will check use them directly
+	*/
+
+	ptpGrandMasterNodeName := os.Getenv("PTP_CONFIG_MASTER_NODE")
```

Comment:
I feel this is not required. This should be native k8s way using by node label ptp/master and ptp/slave.

Reply:
By looking further in the code, we do label the nodes with ptp/master and ptp/slave after deciding which one is master and which one is slave.
what I can do, is assign master to a node with label node-role.kubernetes.io/control-plane or node-role.kubernetes.io/master; but this may not always be reflective of the clock role in the cluster.
I guess the gist of this is to provide developer with an easy way to specify which interfaces to use directly

Reply:
we will leave the node labeling logic as it is since it is expected in that way in the cnf-test repo too.
 If the user is going to specify the interface and he knows the interface nodes. then the user can always label the nodes as ptp/master and ptp/slave as required.